debian/changelog

```diff
@@ -1,3 +1,9 @@
+matrix-synapse-py3 (0.34.0) stretch; urgency=medium
+
+  * New upstream version 0.34.0
```

Comment:
Since we're completely changing the debian packaging, it's probably worth mentioning that here, even though the info is to some extent duplicated in the main changelog.